### DISCUSSION TITLE
Fragmenting molecules, adding linker to specific areas

### DISCUSSION THREAD

**Discussion by orgw** [rdkit/rdkit#4820] [Q&A]
Hi, I'm quite new to RDKit as well as cheminformatics. I want to modify my molecule but don't know how. 

1.I would like to make fragments of my molecule
2. mark where i should link my fragments
3. link my molecule

what functions or ways can i approach this problem?

**Comment by greglandrum** [comment]
Hi @orgw,
There are a lot of ways to interpret your question.
Please provide a specific example of an input molecule, how you would like it to be fragmented, and what the output should be
